com.github.mdh34.quickdocs.json

```diff
@@ -0,0 +1,75 @@
+{
+    "app-id": "com.github.mdh34.quickdocs",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.mdh34.quickdocs",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--device=dri",
+        "--socket=x11",
+        "--socket=wayland",
+        "--talk-name=ca.desrt.dconf",
+        "--filesystem=home",
```

Comment:
Does it need access to home?

Reply:
There are some files in .config but I'll make it more explicit

Reply:
If it's just for config files, there's no need to do anything. By default $XDG_CONFIG_HOME points to place accessible to the application.

Reply:
Ok, I've updated the app to properly use the XDG directories now so this isn't needed :)